server/routers/jobs_router.py

```
@@ -1,145 +1,270 @@
+
+def calcul_salaire(texte_salaire: str, texte_heure: str) -> float:
+    if texte_salaire == None:
+        return None
+    elif texte_salaire[0:7] == "Mensuel":
+        try:
+            try:
+                pattern = r"Mensuel\s+de\s+([\d.,]+)\s*Euros\s+sur\s+([\d.,]+)\s*mois"
+                m = re.search(pattern, texte_salaire, flags=re.IGNORECASE)
+
+                if not m:
+                    raise ValueError("Format non reconnu")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)